src/main/java/core/basesyntax/FigureValues.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureValues {
+    public abstract void printInfo();
```

Comment:
Think about naming closer to the task. Maybe names `Drawable` and `void draw` would be more appropriate?